Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -2621,13 +2622,21 @@ private async Task<bool> PrepareDirectoryOutputsAsync(HashSet<AbsolutePath> pres
                                         }
                                     });
 
+                                bool allOutputsPrivate = true;
+
                                 foreach (var path in filePaths)
                                 {
                                     if (!await m_makeOutputPrivate(path))

```

Comment:
>m_makeOutputPrivate [](start = 47, length = 19)

Depending on how TryMakeExclusiveLinkAsync fails there may be dangling temp files. Should we clean them up?

Reply:
For preserved output file, they will eventually be scrubbed.
For preserved output directory, the directory will be wiped out, and so the dangling temp file should get deleted as well.